app/src/main/java/com/android/voyageur/ui/trip/activities/Activities.kt

```diff
@@ -176,86 +177,96 @@ fun ActivitiesScreen(
         }
       },
       content = { pd ->
-        val isEditable = !isReadOnly
-        val buttonType = if (isEditable) ButtonType.DELETE else ButtonType.NOTHING
-        LazyColumn(
-            modifier = Modifier.padding(pd).fillMaxWidth().testTag("lazyColumn"),
-            verticalArrangement = Arrangement.spacedBy(10.dp, Alignment.Top),
-        ) {
-          item {
+        if (drafts.isEmpty() && final.isEmpty()) {
+          // Display empty prompt if there are no activities
+          Box(modifier = Modifier.padding(pd).fillMaxSize(), contentAlignment = Alignment.Center) {
             Text(
-                text = stringResource(R.string.drafts),
-                fontSize = 24.sp,
-                fontWeight = FontWeight.Bold,
-                modifier = Modifier.padding(start = 10.dp))
+                modifier = Modifier.testTag("emptyActivitiesPrompt"),
+                text = stringResource(R.string.no_activities_scheduled),
+            )
           }
-          drafts.forEach { activity ->
+        } else {
+          val isEditable = !isReadOnly
+          val buttonType = if (isEditable) ButtonType.DELETE else ButtonType.NOTHING
+          LazyColumn(
+              modifier = Modifier.padding(pd).fillMaxWidth().testTag("lazyColumn"),
+              verticalArrangement = Arrangement.spacedBy(10.dp, Alignment.Top),
+          ) {
             item {
-              if (selectedFilters.isEmpty() || activity.activityType in selectedFilters) {
-                ActivityItem(
-                    activity,
-                    isEditable,
-                    onClickButton = {
-                      activityToDelete = activity
-                      showDialog = true
-                    },
-                    buttonType,
-                    navigationActions,
-                    tripsViewModel)
-                Spacer(modifier = Modifier.height(10.dp))
+              Text(
```

Comment:
I was thinking, what if we show "Drafts" only if we currently have draft activities? Same for "Final". That way we avoid having one final activity and the screen showing 
Drafts
Final
Activity

Reply:
Fixed.